src/assets/wise5/directives/teacher-summary-display/summary-data/DialogGuidanceSummaryDataPoint.ts

```diff
@@ -0,0 +1,11 @@
+import { ComponentState } from '../../../../../app/domain/componentState';
+import { IdeasSummaryDataPoint } from './IdeasSummaryDataPoint';
+
+export class DialogGuidanceSummaryDataPoint extends IdeasSummaryDataPoint {
+  constructor(componentState: ComponentState) {
+    super();
+    componentState.studentData.responses.forEach((response) => {
+      response.ideas?.forEach((idea) => this.processIdea(idea));
+    });
```

Comment:
Remove inner brackets
```suggestion
    componentState.studentData.responses.forEach((response) =>
      response.ideas?.forEach((idea) => this.processIdea(idea))
    );
```